packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/WorklistDataGrid.1/Examples.3/WorklistDataGridFocusableCell.doc.mdx

```diff
@@ -1,3 +1,22 @@
 import WorklistDataGridFocusableCell from './WorklistDataGridFocusableCell?dev-site-example';
 
-<WorklistDataGridFocusableCell title= 'Focusable elements within cells' />
+# Single Cell Navigation (Dive-In)
+
+### Description
+
+The content of a cell in the [Worklist DataGrid](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about) may be empty, text, or any combination of widgets. Since the [grid pattern](https://www.w3.org/WAI/ARIA/apg/patterns/grid/) mandates only one tab stop and arrow keys for navigation, these keys are not available by default to navigate focusable elements inside a cell. To make these keys available for cell navigation, as part of its basic functionality, the Worklist DataGrid supports single cell navigation (dive-in mode). Enabling single cell navigation will disable grid navigation and disabling single cell navigation will restore grid navigation.
```

Comment:
```suggestion
The content of a cell in the [Worklist DataGrid](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about) may be empty, text, numerical, or any combination of widgets. Since the [grid pattern](https://www.w3.org/WAI/ARIA/apg/patterns/grid/) mandates only one tab stop and arrow keys for navigation, these keys are not available by default to navigate focusable elements inside a cell. To make these keys available for cell navigation, as part of its basic functionality, the Worklist DataGrid supports single cell navigation (dive-in mode). Enabling single cell navigation will disable grid navigation and disabling single cell navigation will restore grid navigation.
```

Reply:
Updated